wowchemy-cms/assets/js/cms-preview-templates/post.js

```diff
@@ -0,0 +1,28 @@
+let PostPreview = createClass({
+  render: function() {
+    let entry = this.props.entry;
+    let bg = this.props.getAsset('featured.jpg')
+    if (bg == 'featured.jpg') { // bg will be equal to 'featured.jpg' if the image doesn't exist
```

Comment:
Because the featured image can have a .jpg/.jpeg/.png file extension, we need to check all three options.
The `getAsset()` function returns back the filename if the file doesn't exist.

Reply:
@paazca these are the unprocessed image though right? So it could try to download large (e.g. 10MB) image files for the preview? Unless `getAsset` is optimizing them?

Reply:
I don't believe that `getAsset` optimizes the images, but (if I've understood you correctly) that's no different than the regular image preview in the CMS.